server/routes/index.ts

```diff
@@ -19,5 +20,8 @@ export default function routes(services: Services): Router {
   const temporaryAbsencesController = new TemporaryAbsencesController(services.temporaryAbsencesService)
   get('/confirm-arrival/return-from-temporary-absence', temporaryAbsencesController.view())
 
+  const imageController = new ImageController(services.incomingMovementsService)
```

Comment:
could maybe call it prisoner or offender controller? 

Image is quite generic to be sitting on such a specific url

Reply:
renamed to prisonerController